pyproject.toml

```diff
@@ -113,7 +107,7 @@ supported_nautobot_versions = [
 
 [tool.ruff]
 line-length = 120
-target-version = "py38"
+target-version = "py313"
```

Comment:
should be py3.12?  Otherwise currently the python dependency is not allowing 3.13 to be installed.

Reply:
Per the docs https://docs.astral.sh/ruff/settings/#target-version, this should be the minimum supported version by the project as the auto-format could change the code to be incompatible with earlier versions of python.